app/controllers/api/v1/api_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Api::V1::ApiController < ActionController::API
+  before_action :set_default_format
+  before_action :authenticate_token!
+
+   private
+
+     def set_default_format
+      request.format = :json
```

Comment:
why are we setting the format here ?

Reply:
Sir my project app. supports json API that's why I am setting format.